src/tools/pub/package-layout.md

```diff
@@ -67,13 +67,13 @@ enchilada/
     [application package](/tools/pub/glossary#application-package).
 
 \*** The `doc/api` directory exists locally after you've run
-      [dartdoc](https://github.com/dart-lang/dartdoc#dartdoc).
+      [dartdoc.](https://github.com/dart-lang/dartdoc#dartdoc)
```

Comment:
Really? I'd think you want the dot outside the link?

Reply:
We have a little external-link indicator that looks terrible when it's inside the punctuation.

https://github.com/dart-lang/site-angulardart/wiki/Why-does-external-link-text-contain-punctuation%3F

Reply:
Oh... 🤷‍♂